controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const myAddress = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myAddressShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addressPatch = async (req, res) => {
+  try {
+    await userService.myAddressPatch(req);
```

Comment:
```suggestion
    await userService.updateUserAddress( . . .);
```
해당 api 또한 userId가 필요하겠네요.  또한 body에 담긴 수정된  address를 가져와 변수 할당 한 뒤 넘겨주는 식으로 바꿔주세요! 말씀드리는 코드가 controller가 아닌 service layer에 작성되어있습니다.

Reply:
네 기능에맞게 레이어 전부 수정했고, 파라미터 역시 수정했습니다 !